api/stampy_client.py

```diff
@@ -0,0 +1,859 @@
+#!/usr/bin/env python3
```

Comment:
I don't like having scripts like this commited - they tend to rot. Isn't the frontend sufficient for general testing? i.e. I'm all for you having your own scripts - this is the way - but dunno about commiting them